### PR TITLE
VEP 99.2 foss 2019b

### PR DESCRIPTION
**Pull request by afraz-ahmed**
Needs #107 

For INC1019168

VEP-99.2-foss-2019b-Perl-5.30.0.eb
* [x] Assigned to reviewer
* [ ] EL7-cascadelake
* [ ] EL7-haswell
* [ ] EL7-sandybridge
* [ ] Ubuntu16 VM

### CODE REVIEW

**Comment by branfosj** [comment]
We'll need to alter this from the upstream version - one the people who uses VEP always wants an a specific extra plugin to work and that involves an addition: https://gitlab.bham.ac.uk/res-comp-team/bear-software/2019a/easybuild-easyconfigs/-/blob/master/easybuild/easyconfigs/v/VEP/VEP-96.0-foss-2019a-Perl-5.28.1.eb

**Comment by afraz-ahmed** [comment]
> We'll need to alter this from the upstream version - one the people who uses VEP always wants an a specific extra plugin to work and that involves an addition: https://gitlab.bham.ac.uk/res-comp-team/bear-software/2019a/easybuild-easyconfigs/-/blob/master/easybuild/easyconfigs/v/VEP/VEP-96.0-foss-2019a-Perl-5.28.1.eb

Hi, Simon. I have added the extra files and updated all dependencies. Thanks.

File: easybuild/easyconfigs/b/Bio-DB-BigFile-jksrc/Bio-DB-BigFile-jksrc-v395-foss-2019b.eb [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# This easyconfig was created by the BEAR Software team at the University of Birmingham.
+easyblock = 'CmdCp'
+
+name = 'Bio-DB-BigFile-jksrc'
+version = 'v395'
+
+homepage = "http://hgdownload.cse.ucsc.edu/admin/"
```
Comment (branfosj) [inline easybuild/easyconfigs/b/Bio-DB-BigFile-jksrc/Bio-DB-BigFile-jksrc-v395-foss-2019b.eb:7]:
```suggestion
homepage = "https://hgdownload.cse.ucsc.edu/admin/"
```

File: easybuild/easyconfigs/b/Bio-DB-BigFile-jksrc/Bio-DB-BigFile-jksrc-v395-foss-2019b.eb [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+description = """A full copy of the 'kent' source tree - used here for only one dependency."""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://hgdownload.cse.ucsc.edu/admin/']
```
Comment (branfosj) [inline easybuild/easyconfigs/b/Bio-DB-BigFile-jksrc/Bio-DB-BigFile-jksrc-v395-foss-2019b.eb:13]:
```suggestion
source_urls = ['https://hgdownload.cse.ucsc.edu/admin/']
```

File: easybuild/easyconfigs/l/LibUUID/LibUUID-1.0.3-GCCcore-8.3.0.eb [hunk 3]
```diff
@@ -0,0 +1,40 @@
+easyblock = 'ConfigureMake'
+
+name = 'LibUUID'
+version = '1.0.3'
+
+homepage = 'http://sourceforge.net/projects/libuuid/'
```
Comment (branfosj) [inline easybuild/easyconfigs/l/LibUUID/LibUUID-1.0.3-GCCcore-8.3.0.eb:6]:
```suggestion
homepage = 'https://sourceforge.net/projects/libuuid/'
```

File: easybuild/easyconfigs/b/Bio-DB-BigFile-jksrc/Bio-DB-BigFile-jksrc-v395-foss-2019b.eb [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+source_urls = ['https://hgdownload.cse.ucsc.edu/admin/']
+sources = ['jksrc.%(version)s.zip']
+patches = ['%(name)s-%(version)s.patch']
+checksums = [
+    '9fdde7937e47e96ac1580022bf83ac6c1e38cf89ee2df3bd7237ffcfd673ae19',  # jksrc.v395.zip
+    '8d02f082b16e1e4b17374819d2670f2d8602c7cce6630bedc81f18a101242889',  # Bio-DB-BigFile-jksrc-v395.patch
+]
+
+dependencies = [
+    ('BioPerl', '1.7.2', '-Perl-5.30.0'),
```
Comment (branfosj) [inline easybuild/easyconfigs/b/Bio-DB-BigFile-jksrc/Bio-DB-BigFile-jksrc-v395-foss-2019b.eb:22]:
```suggestion
    ('BioPerl', '1.7.2'),
```

File: easybuild/easyconfigs/b/Bio-DB-BigFile/Bio-DB-BigFile-1.07-foss-2019b-Perl-5.30.0.eb [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+sources = ['Bio-BigFile-%(version)s.tar.gz']
+patches = ['%(name)s-%(version)s.patch']
+checksums = [
+    '277b66ce8acbdd52399e2c5a0cf4e3bd5c74c12b94877cd383d0c4c97740d16d',  # Bio-BigFile-1.07.tar.gz
+    '43187145915138360006324ab084fd5eff005e9c4441362a13446339aba527f5',  # Bio-DB-BigFile-1.07.patch
+]
+
+dependencies = [
+    ('Perl', '5.30.0'),
+    ('BioPerl', '1.7.2', versionsuffix),
```
Comment (branfosj) [inline easybuild/easyconfigs/b/Bio-DB-BigFile/Bio-DB-BigFile-1.07-foss-2019b-Perl-5.30.0.eb:22]:
```suggestion
    ('BioPerl', '1.7.2'),
```

File: easybuild/easyconfigs/v/VEP/VEP-99.2-foss-2019b-Perl-5.30.0.eb [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+source_urls = ['https://github.com/Ensembl/ensembl-vep/archive/release/']
+sources = ['%(version)s.tar.gz']
+checksums = ['4488a5efe1047d489f1d1391d9e164cc159d0e9912e593b88140da279d0875cb']
+
+dependencies = [
+    ('Perl', '5.30.0'),
+    ('DBD-mysql', '4.050', versionsuffix),
+    ('BioPerl', '1.7.2', versionsuffix),
```
Comment (branfosj) [inline easybuild/easyconfigs/v/VEP/VEP-99.2-foss-2019b-Perl-5.30.0.eb:21]:
```suggestion
    ('BioPerl', '1.7.2'),
```

**Comment by edmondac** [comment]
Build failed with:
```
Can't locate ExtUtils/CppGuess.pm in @INC (you may need to install the ExtUtils::CppGuess module 
```

**Pull request merged**